### ISSUE TITLE
Database Viewer help doesn't say how to run it.

### ISSUE THREAD

**Issue by gwideman** [NationalSecurityAgency/ghidra#1988]
**Describe the bug**
Ghidra has a help page titled "Database Viewer", describing it as "The Database Viewer displays the raw database tables used in the Ghidra program database."

But it doesn't say how to actually run this viewer. I have searched Ghidra's menus and not found it.

**To Reproduce**
1. Open Help
2. Use Find with string "Database Viewer" 
3. Click result "Comments"
4. Read description and admire the screen cap.
5. Experience bewilderment at not knowing how to invoke this tool

**Expected behavior**
I expected help to provide a description of how to invoke the Database Viewer.

**Screenshots**
Ghidra's help page for Database Viewer
![gw_databaseviewer](https://user-images.githubusercontent.com/1243021/84587351-54c32000-add3-11ea-9ef5-7097c00e1132.png)

**Attachments**
[none]

**Environment (please complete the following information):**
 - OS: Win10-64
 - Java Version: 11.0.7
 - Ghidra Version: 9.1.2

**Additional context**
Hoping to examine more closely what Ghidra does and doesn't know and store as analysis proceeds.

**Comment by saruman9** [comment]
In menu bar: `File` → `Configure` → `Developer` (press `Configure`) → `DbViewerPlugin` (check flag).
Database Viewer will be in `Window` top menu.
The plugin disabled by default, because it's a debug plugin. Information about enabling debug plugins is unnecessary information in the help page IMHO.

**Comment by gwideman** [comment]
@saruman9 Thanks very much for your quick response. Very useful!

To address your other point: If the Help shouldn't say how to enable the Database Viewer plugin, then why does it have any info on this plugin at all?

And it's very lucky that it _does_ have info on the Database Viewer, because otherwise I never would have learned about it, as it's pretty much invisible to google. Slightly less so now with this thread :-).

Yet it's extremely informative, because, as I anticipated in my original post, it exposes what Ghidra knows, unobscured by UI. So it gives users a second way to understand what ghidra is doing, which gives them a fighting chance where the docs are cloudy or absent.

In short -- it's not at all just a developer tool!

**Comment by dragonmacher** [comment]
The DB Viewer is typically only used by those writing database tables or tracking down esoteric bugs.   It certainly can be used by anyone wishing to see the tables directly.   In practice, this is very few users.

I think the real issue here is that we do not universally indicate how to active a given plugin in its help documentation.  Ideally, each help page would list how to access a given feature, including how to load a plugin if it is not loaded.
